File: airbyte-commons/src/main/java/io/airbyte/commons/constants/AirbyteSecretConstants.java
/*
 * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.constants;

/**
 * Collection of constants related to Airbyte secrets defined in connector configurations.
 */
public final class AirbyteSecretConstants {

  /**
   * The name of a configuration property field that has been identified as a secret.
   */
  public static final String AIRBYTE_SECRET_FIELD = "airbyte_secret";

  /**
   * Mask value that is displayed in place of a value associated with an airbyte secret.
   */
  public static final String SECRETS_MASK = "**********";

  private AirbyteSecretConstants() {
    // Private constructor to prevent instantiation
  }

}

File: airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/constants/AirbyteSecretConstants.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.commons.constants

/** Collection of constants related to Airbyte secrets defined in connector configurations. */
object AirbyteSecretConstants {
    /** The name of a configuration property field that has been identified as a secret. */
    const val AIRBYTE_SECRET_FIELD: String = "airbyte_secret"

    /** Mask value that is displayed in place of a value associated with an airbyte secret. */
    const val SECRETS_MASK: String = "**********"
}
